src/app/(sidebar)/account/muxed-create/page.tsx

```diff
@@ -1,5 +1,194 @@
+
+  const [isReset, setReset] = useState<boolean>(false);
+
+  const generateMuxedAccount = () => {
+    const result = muxedAccount.generate({
+      baseAddress,
+      muxedAccountId: muxedId,
+    });
+
+    const { error, muxedAddress } = result;
+
+    if (muxedAddress) {
+      setReset(false);
+      account.updateGeneratedMuxedAccount({
+        id: muxedId,
+        baseAddress,
+        muxedAddress,
+      });
+
+      setSdkError("");
+      return;
+    }
+
+    if (error) {
+      setSdkError(error);
+      return;
+    }
+  };
+
+  return (
+    <div className="Account">
+      <Card>
+        <div className="Account__content">
+          <div className="Account__card">
+            <div className="CardText">
+              <Text size="lg" as="h1" weight="medium">
+                Create Multiplexed Account
+              </Text>
+
+              <Text size="sm" as="p">
+                A muxed (or multiplexed) account (defined in{" "}
+                <SdsLink href="https://github.com/stellar/stellar-protocol/blob/master/core/cap-0027.md">
+                  CAP-27
+                </SdsLink>{" "}
+                and briefly{" "}
+                <SdsLink href="https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0023.md">
+                  SEP-23
+                </SdsLink>
+                ) is one that resolves a single Stellar G...account to many
+                different underlying IDs.
+              </Text>
+            </div>
+
+            <PubKeyPicker
+              id="muxed-public-key"
+              label="Base Account G Address"
+              value={baseAddress}
+              onChange={(e) => {
+                setReset(true);
+                setBaseAddress(e.target.value);
+
+                const error = validate.publicKey(e.target.value);
+
+                setBaseFieldErrorMessage(error || "");
+              }}
+              error={baseFieldErrorMessage}
+              copyButton={{
+                position: "right",
+              }}
+            />
+
+            <MuxedIdPicker
+              id="muxed-account-id"
+              label="Muxed Account Id"
+              value={muxedId}
+              onChange={(e) => {
+                setReset(true);
+                setMuxedId(e.target.value);
+
+                const error = validate.positiveInt(e.target.value);
+                setMuxedFieldError(error || "");
+              }}
+              error={muxedFieldError}
+              copyButton={{
+                position: "right",
+              }}
+            />
+
+            <div className="Account__CTA">
+              <Button
+                disabled={!baseAddress || Boolean(baseFieldErrorMessage)}
+                size="md"
+                variant={"secondary"}
+                onClick={generateMuxedAccount}
+              >
+                Create
+              </Button>
+            </div>
+          </div>
+
+          <ExpandBox
```

Comment:
We might need to put the `ExpandBox` inside another container because it's creating extra space at the bottom where it is now. We can do something similar to Create Account.

![image](https://github.com/stellar/laboratory/assets/7346473/d0c0ad57-d2b6-4197-979d-e53a2ff4f279)

Reply:
eagle eyes!!!

Reply:
same thing happened in a different container. Create Account is using `margin-top` inside the div. Not sure how I can achieve this using `gap`. it seems like they don't collapse even when a div is not there.

Reply:
Ooh, I missed the `margin-top` in Create Account. We shouldn't use margins for layout spacing. I'll create a component that will make this easier.